roles/kustomize_deploy/tasks/01_install_operators.yml

```diff
@@ -0,0 +1,138 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Kustomize and deploy OSP operators
+  block:
+    - name: Set the kustomization output filename
+      ansible.builtin.set_fact:
+        _cifmw_kustomize_deploy_olm_dest_file: >-
+          {{
+            [
+              cifmw_kustomize_deploy_kustomizations_dest_dir,
+              'olm.yml'
+            ] | path_join
+          }}
+        _cifmw_kustomize_deploy_olm_source_files: >-
+          {{
+            [
+              cifmw_kustomize_deploy_architecture_repo_dest_dir,
+              'examples',
+              'common',
+              'olm'
+            ] | path_join
+          }}
+
+    - name: Perform kustomization on the OLM directory
+      cifmw.general.ci_kustomize:
```

Comment:
we should be 100% transparent here, and not use ci_kustomize (that module may even be dropped imho).
We should use ci_script with relevant `oc` or other commands, to ensure it's readable, and devs may even jump in when there's an update on this specific topic.

Reply:
replaced with `kubernetes.core.kustomize` plugin